util/src/main/scala/org/ensime/util/io.scala

```diff
@@ -3,15 +3,27 @@
 package org.ensime.util
 
 import java.io._
-import com.google.common.io.ByteStreams
 
 /**
  * NOTE: prefer NIO via the path utilities.
  */
 package object io {
 
   implicit class RichInputStream(val is: InputStream) extends AnyVal {
-    def toByteArray(): Array[Byte] = ByteStreams.toByteArray(is)
+    def toByteArray(): Array[Byte] = {
+      val baos = new ByteArrayOutputStream()
+
+      val data = Array.ofDim[Byte](16384)
+      var nRead = is.read(data, 0, data.length)
+
+      while (nRead != -1) {
+        baos.write(data, 0, nRead)
+        nRead = is.read(data, 0, data.length)
+      }
+
+      baos.flush()
+      baos.toByteArray()
```

Comment:
doesn't look like we're closing the input at the end, does that agree with the guava semantics?